### PR TITLE
Update sentry node docs

### LINKED ISSUE #539: Update Sentry Guide
https://wiki.polkadot.network/docs/en/maintain-guides-how-to-setup-sentry-node

There are some flags that need to be updated:
- the addition of the `--sentry-nodes` flag
- `--sentry` and `--sentry-nodes` imply reserved nodes
- Add notes on authority discovery

### PR DESCRIPTION
**Pull request by wpank**
- updates sentry guide to include details on libp2p networking and updated flags

closes #539

### CODE REVIEW

**Comment by wheresaddie** [comment]
really nice work @wpank -- documentation is easier to parse and reference now.. thanks!

File: docs/maintain-guides-how-to-setup-sentry-node.md [hunk 1]
```diff
@@ -152,80 +152,161 @@ You can then use `ping` to verify the connectivity between the nodes.
...
 
 After you have started the `wg0` interface on your public node and validator, do spend a little bit
 of time to take a look at the following description of those flags you are going to use.
 
-`--sentry` - This would be required for your public node to be an authority as an observer. That
-means it acts the same as a validator node but without holding keys / signing. And the difference
-between running a full node versus adding an extra `--sentry` flag is that a full node might not
-have all the data the validator needs to validate properly.
+`--sentry <VALIDATOR_MULTIADDR>` - This would be required for your public node to be an authority as
+an observer. That means it acts the same as a validator node but without holding keys / signing. And
```
Comment (lsaether) [inline docs/maintain-guides-how-to-setup-sentry-node.md]:
```suggestion
an observer. That means it acts the same as a validator node but without holding keys or signing.
```

File: docs/maintain-guides-how-to-setup-sentry-node.md [hunk 2]
```diff
@@ -152,80 +152,161 @@ You can then use `ping` to verify the connectivity between the nodes.
...
+the difference between running a full node versus adding an extra `--sentry` flag is that a full
```
Comment (lsaether) [inline docs/maintain-guides-how-to-setup-sentry-node.md]:
```suggestion
The difference between running a full node versus adding an extra `--sentry` flag is that a full
```

File: docs/maintain-guides-how-to-setup-sentry-node.md [hunk 3]
```diff
@@ -152,80 +152,161 @@ You can then use `ping` to verify the connectivity between the nodes.
...
+- `P2P_PORT` - This is the port that nodes will send p2p messages over. By default, this will be
+  30333, but can be explicitly specified using the `--port <P2P_PORT>` cli flag.
+
+- `PEER IDENTITY` - The PeerId is a unique identifier for each peer.
+
+##### PeerId
+
+Each peer in the network will have a private (secret) key (not to be confused with account keys or
+session keys) that will be used for network messaging. The secret key will have a corresponding
+public key, as well as a public address (`PeerId`) derived from this private key. `PeerId`'s are
```
Comment (lsaether) [inline docs/maintain-guides-how-to-setup-sentry-node.md]:
```suggestion
public key, as well as a public address (`PeerId`) derived from this private key. `PeerId` is
```

File: docs/maintain-guides-how-to-setup-sentry-node.md [hunk 4]
```diff
@@ -152,80 +152,161 @@ You can then use `ping` to verify the connectivity between the nodes.
...
+specified), a new `secret_ed25519` will be created and used to derive a new `PeerId`.
+
+If you want to explicitly specify the secret key to use, you can do so with the
+`--node-key-file <KEY_FILE>` flag, where `KEY_FILE` is the path of a file containing an unencoded 32
+byte Ed25519 secret key, or the `--node-key <KEY>` flag, where `KEY` is a hex-encoded 32 byte secret
+key. If explicitly specifying a secret key, it is recommended to specify it as a file.
 
+> Note: It is useful to generate or back up the `secret_ed25519` files if you want to use static
+> addresses for sentry configurations. Otherwise you may have to dynamically find and set
+> `PeerId`'s.
```
Comment (lsaether) [inline docs/maintain-guides-how-to-setup-sentry-node.md]:
```suggestion
> the `PeerId`.
```

prefer not using the apostrophe "s" on code tags

**Review by lsaether — CHANGES_REQUESTED** [review]
nits

**Review by wheresaddie — APPROVED** [review]

**Review by lsaether — APPROVED** [review]

**Pull request merged**